cachi2/core/package_managers/generic/main.py

```diff
@@ -72,7 +113,7 @@ def _load_lockfile(lockfile_path: RootedPath) -> GenericLockfileV1:
         except ValidationError as e:
             loc = e.errors()[0]["loc"]
             msg = e.errors()[0]["msg"]
-            raise PackageManagerError(
+            raise PackageRejected(
```

Comment:
nitpick: This should be a separate commit as it's a unrelated change fixing a wrong type of error used for the given scenario.

Reply:
Separated into new commit